ui/common/src/main/java/de/danoeh/antennapod/ui/common/PagedToolbarFragment.java

```diff
@@ -22,12 +21,24 @@ protected void setupPagedToolbar(final MaterialToolbar toolbar, final ViewPager2
             }
             return false;
         });
+        // Defer findFragmentByTag until onPageScrollStateChanged (SCROLL_STATE_IDLE), as the FragmentManager
+        // isn’t ready on the first call to onPageSelected (when first switching tabs).
```

Comment:
Could you please mention this in the commit message instead? Most of the remaining code explains things like these in the commit messages instead of code comments.